crates/ironrdp-rdpdr/src/backend/mod.rs

```diff
@@ -7,10 +7,11 @@ use ironrdp_svc::AsAny;
 
 use crate::pdu::efs::{DeviceControlRequest, ServerDeviceAnnounceResponse, ServerDriveIoRequest};
 use crate::pdu::esc::{ScardCall, ScardIoCtlCode};
+use ironrdp_svc::SvcMessage;
 
 /// OS-specific device redirection backend interface.
 pub trait RdpdrBackend: AsAny + fmt::Debug + Send {
     fn handle_server_device_announce_response(&mut self, pdu: ServerDeviceAnnounceResponse) -> PduResult<()>;
     fn handle_scard_call(&mut self, req: DeviceControlRequest<ScardIoCtlCode>, call: ScardCall) -> PduResult<()>;
-    fn handle_drive_io_request(&mut self, req: ServerDriveIoRequest) -> PduResult<()>;
+    fn handle_drive_io_request(&mut self, req: ServerDriveIoRequest) -> PduResult<Vec<SvcMessage>>;
```

Comment:
@ibeckermayer This is slightly changing the API for `RdpdrBackend`. Does that look good to you?